### PR TITLE
Add infinity play option

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolve: [#91 ](https://github.com/VolkovLabs/volkovlabs-image-panel/issues/91)

### CODE REVIEW

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

File: src/components/ImagePanel/ImagePanel.tsx [hunk 1]
```diff
@@ -289,9 +289,11 @@ export const ImagePanel: React.FC<Props> = ({ options, data, width, height, repl
   if (type === SupportedFileType.MP4 || type === SupportedFileType.WEBM) {
     return renderContainer(
       <video
+        muted={options.autoPlay}
```
Comment (mikhail-vl) [inline src/components/ImagePanel/ImagePanel.tsx:292]: Why do we need it?

Comment (asimonok) [reply src/components/ImagePanel/ImagePanel.tsx:292]: Browsers require muted attribute enabled to auto play the video - https://developer.chrome.com/blog/autoplay#new_behaviors

> Chrome's autoplay policies are simple:
> 
> Muted autoplay is always allowed.
> Autoplay with sound is allowed if:
> The user has interacted with the domain (click, tap, etc.).
> On desktop, the user's [Media Engagement Index](https://developer.chrome.com/blog/autoplay#media_engagement_index) threshold has been crossed, meaning the user has previously played video with sound.
> The user has [added the site to their home screen](https://web.dev/customize-install/) on mobile or [installed the PWA](https://web.dev/progressive-web-apps/) on desktop.
> Top frames can [delegate autoplay permission](https://developer.chrome.com/blog/autoplay#iframe_delegation) to their iframes to allow autoplay with sound.

Comment (mikhail-vl) [reply src/components/ImagePanel/ImagePanel.tsx:292]: Please add description to mention it.

**Pull request merged**